ledger-core/instrumentation/insapp/server.go

```diff
@@ -181,3 +184,7 @@ func (s *Server) Stop() {
 	s.gracefulStop <- syscall.SIGQUIT
 	<-s.waitChannel
 }
+
+func (s *Server) Started() bool {
```

Comment:
I think it's better to have WaitStarted with channel, then have bool function with sleeps